### DISCUSSION TITLE
I get -65535 score

### DISCUSSION THREAD

**Discussion by zhanshijinwat** [PaddlePaddle/PaddleSpatial#116] [BaiduKDDCup2022] [answered]
I update evalutaion.py and metrics.py to latest version and I can get 46 score in my computer. However, I get -65535 score after submit. There are't nan in my prediction. How can I fix it?

**Comment by ramber1836** [comment]
NaN prediction, empty prediction, and the prediction with (almost) identical values are treated as invalid. Besides, irregular Accuracy values are also treated as an abnormal prediction.

**Reply by Legend94rz** [reply]
> with (almost) identical values are treated as invalid.

What is `almost` identical values? How to check?

**Reply by zhanshijinwat** [reply]
> 

But It is normal in my computer（I can get 46 score），only abnormal after submitted.  The evaluation.py and metrics.py on github is the latest version?  Beside, test_X and test_Y which I download is same to the which on the server?  Maybe because of the difference of dataset?

**Comment by zhanshijinwat** [comment]
Maybe the rmse is too big and acc<0?

**Comment by ramber1836** [comment, marked as answer]
A more detailed error msg rather than an abnormal score is provided.
Please try it again to see what is going wrong there.
